app/components/primer/alpha/overlay.pcss

```diff
@@ -3,13 +3,21 @@
   padding: 0;
   position: absolute;
   min-width: 192px;
+  inset: auto;
 }
 
-/* stylelint-disable-next-line selector-pseudo-class-no-unknown */
-.Overlay[popover]:not(:popover-open) {
+.Overlay[popover]:not(.\:popover-open) {
   display: none;
 }
 
-anchored-position {
-  display: block;
+@supports selector(:popover-open) {
+  .Overlay[popover]:not(.\:popover-open) {
+    display: revert;
+  }
+}
+
+@supports selector(:open) {
+  .Overlay[popover]:not(.\:popover-open) {
```

Comment:
Should this be?

```suggestion
  .Overlay[popover]:not(.\:open) {
```

Reply:
It should not. This is specifically _reverting_ the `display` property, to revert lines 9-11

Reply:
It might be nice to add that as a comment above the declaration @keithamus just for more clarity :D